CHANGES.md

```diff
@@ -1,5 +1,7 @@
-unreleased
-----------
+0.21.1 (09/06/2021)
```

Comment:
I would leave this as `unreleased` and replace it only when you actually go through the release. It's okay to push directly on the 0.21 branch to update the changelog's latest entry before cutting the release.

Reply:
Ok, thanks, I'll do it like that for 0.22!